src/domain/app/i18n/sv.json

```diff
@@ -1835,6 +1851,10 @@
       "labelSignupExtraInfo": "Ytterligare information (valfritt)",
       "labelStreetAddress": "Gatuadress",
       "labelZipcode": "Postnummer",
+      "notificationSignupDeleted": "Registreringen har raderats",
```

Comment:
Should it say `Deltagarinformation` instead of `Registeringen` here?

Reply:
Good catch. Fixed in latest commit